.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+# Summary
+
+# Background
+
+# Checklist
+
+- [ ] Added Screenshots
+- [ ] PR linked to issue
```

Comment:
These should be automatic if you create the branch from the issue, but adding here for now as a reminder. There might be a way to automate this, ie fail build if the link doesn't' exist, but not a biggie for me now.